src/Splunk.OpenTelemetry.AutoInstrumentation/nuget-readme.md

```diff
@@ -0,0 +1,82 @@
+# Using the Splunk.OpenTelemetry.AutoInstrumentation NuGet package
+
+## When to use the NuGet package
+
+Use the NuGet package in the following scenarios:
+
+1. Simplify deployment. For example, a container running a single application.
```

Comment:
"Simplify deployment" is a bit subjective. Why wouldn't a developer want to use this instead of the installer MSI or script? Consider removing or rewriting this unless NuGet packages are meant to be the primary method going forward.

Reply:
Agreed, this is subjective. The objective case here is when the developer doesn't have control over the computer image where the application is running, example: developer ships the app but other team is in charge of the container. For such cases the NuGet allow the developer to ship the instrumentation.

Some also consider it more "convenient" to do it via the NuGet package, however, that is subjective too.

I will try to re-word it to avoid the subjective aspect.

Reply:
04122caa2835825cf789537c070d852f73cb0e22